en/integrations/plugin-packs/procedures/applications-oracle-ovm-api.md

```diff
@@ -0,0 +1,175 @@
+
+<!--Offline IMP License-->
+
+1. Install the Centreon Plugin on every Poller:
+
+```bash
+yum install centreon-plugin-Applications-Oracle-Ovm-Api
+```
+
+2. On the Centreon Central server, install the Centreon Plugin-Pack from the RPM:
+
+```bash
+yum install centreon-pack-applications-oracle-ovm-api
+```
+
+3. On the Centreon Web interface in "Configuration > Plugin packs > Manager", install the *Oracle VM Manager API* Plugin-Pack
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Host configuration 
+
+* Add a new Host and apply the *App-Oracle-Ovm-Api-custom* Host Template
+
+> Once the template applied, some Macros have to be configured:
+
+| Mandatory | Name               | Description                                                                |
+| :-------- | :----------------- | :------------------------------------------------------------------------- |
+| X         | OVMAPICUSTOMMODE   | API mode (Default: 'rest')                                                 |
+| X         | OVMAPIPORT         | Port used (Default: 7002)                                                  |
+| X         | OVMAPIPROTO        | Specify https if needed (Default: 'https')                                 |
+| X         | OVMAPIPASSWORD     | Oracle VM Manager username                                                 |
+| X         | OVMAPIUSERNAME     | Oracle VM Manager password                                                 |
+|           | OVMAPIEXTRAOPTIONS | Any extra option you may want to add to the command (eg. a --verbose flag) |
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test by running the following command
+(Parameters such as ```api-username``` or ```api-password```have to be adjusted):
+
+```bash
+/usr/lib/centreon/plugins/centreon_oracle_ovm_api.pl \
+    --plugin=apps::oracle::ovm::api::plugin \
+    --mode=server-pools \
+    --hostname='10.30.2.79' \
+    --port='7002' \
+    --proto='https' \
+    --api-username='myapiusername' \
+    --api-password='myapipassword' \
+    --filter-server-pool-name='TESTPOOL01' \
+    --warning-memory-usage-prct='90' \
+    --critical-memory-usage-prct='95' \
+    --verbose
+OK: Server pool 'TESTPOOL01' servers running: 2, stopped: 0 - virtual machines running: 3, stopped: 0 - memory usage total: 255.32 GB used: 217.87 GB (85.33%) free: 37.45 GB (14.67%) | 'TESTPOOL01#serverpool.servers.running.count'=2;;;0;2 'TESTPOOL01#serverpool.servers.stopped.count'=0;;;0;2 'TESTPOOL01#serverpool.vm.running.count'=3;;;0;3 'TESTPOOL01#serverpool.vm.stopped.count'=0;;;0;3 'TESTPOOL01#serverpool.memory.usage.bytes'=233939402752B;;;0;274148098048 'TESTPOOL01#serverpool.memory.free.bytes'=40208695296B;;;0;274148098048 'TESTPOOL01#serverpool.memory.usage.percentage'=85.33%;90;95;0;100
```

Comment:
It's better to have separate CLI and output to make copy/paste easier for a end user. 

-- verbose
\```

ADD => Which outputs something similar to: 

\```
OK: ....